docs/conf.py

```diff
@@ -291,6 +304,7 @@ def get_current_years():
     ("css/popupStyles.css", {"priority": 999}),
     ("css/custom_styles.css", {"priority": 998}),
     "css/colors.css",
+    'https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.0.0-beta3/css/all.min.css'
```

Comment:
this shouldn't be necessary, we already rely on `fontawesome`, please try without. Or discuss with me.